Resources/public/js/views/services/ez-contenteditviewservice.js

```diff
@@ -400,6 +400,10 @@ YUI.add('ez-contenteditviewservice', function (Y) {
         _redirectToAttribute: function (attr) {
             var attrRedirectionUrl = this.get(attr);
 
+            this.set('publishRedirectionUrl');
```

Comment:
The idea behind this is to reset (somehow) the attribute value. Using `this.reset('publishRedirectionUrl');` didn't work correctly. I found it many times, thus that's why I'm setting undefined here.
Without reseting the values of these attributes a user might be unexpectedly redirected somewhere else after creating content. I found this as an issue when trying to redirect a user to the dashboard after editing content from dashboard block.